src/components/Confirm/index.js

```diff
@@ -60,7 +60,7 @@ const Previews = localised(({capture, retakeAction, confirmAction, error, method
     <div className={classNames(style.previewsContainer, theme.fullHeightContainer, {
       [style.previewsContainerIsFullScreen]: isFullScreen,
     })}>
-      { error.type ? <Error {...{error, withArrow: true}} /> :
+      { error.type ? <Error {...{error, withArrow: true, role: "alertdialog", focusOnRender: true}} /> :
```

Comment:
Just to note that the role has to be "alert" when there is not interaction, e.g. error popup cannot be closed, otherwise the ones with interaction are "alertdialog".

Reply:
The main reason why I decided to use "alertdialog" is because this whole screen in a way behaves as an alert dialog where the only thing you can do is click on "Redo". The second reason is that "alert" is always read twice by the screen reader, which seems a bit invasive and not great in terms of UX. Having said that, I've tested a few alert examples and they are all read twice, so probably that's the expected behaviour.

Reply:
I will change it to alert as the that seem to be the correct usage and behaviour